x-pack/plugins/data_enhanced/public/search/sessions_mgmt/application/index.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { CoreSetup } from 'kibana/public';
+import type { ManagementAppMountParams } from 'src/plugins/management/public';
+import type {
+  AppDependencies,
+  IManagementSectionsPluginsSetup,
+  IManagementSectionsPluginsStart,
+  SessionsMgmtConfigSchema,
+} from '../';
+import { APP } from '../';
+import { SearchSessionsMgmtAPI } from '../lib/api';
+import { AsyncSearchIntroDocumentation } from '../lib/documentation';
+import { renderApp } from './render';
+
+export class SearchSessionsMgmtApp {
+  constructor(
+    private coreSetup: CoreSetup<IManagementSectionsPluginsStart>,
+    private config: SessionsMgmtConfigSchema,
+    private params: ManagementAppMountParams,
+    private pluginsSetup: IManagementSectionsPluginsSetup
+  ) {}
+
+  public async mountManagementSection() {
+    const { coreSetup, params, pluginsSetup } = this;
+    const [coreStart, pluginsStart] = await coreSetup.getStartServices();
+
+    const {
+      chrome: { docTitle },
+      http,
+      docLinks,
+      i18n,
+      notifications,
+      uiSettings,
+    } = coreStart;
+    const { data, share } = pluginsStart;
+
+    const pluginName = APP.getI18nName();
+    docTitle.change(pluginName);
+    params.setBreadcrumbs([{ text: pluginName }]);
+
+    const { sessionsClient } = data.search;
+    const api = new SearchSessionsMgmtAPI(
+      sessionsClient,
+      share.urlGenerators,
+      notifications,
+      this.config
+    );
+
+    const documentation = new AsyncSearchIntroDocumentation(docLinks);
+
+    const dependencies: AppDependencies = {
+      plugins: pluginsSetup,
+      config: this.config,
+      documentation,
+      core: coreStart,
+      api,
+      http,
+      i18n,
+      uiSettings,
+      share,
+    };
+    const initialTable = await api.fetchTableData();
```

Comment:
I think we should remove it and let table handle this initial request instead. 
Otherwise, I assume this leads to the blank screen while this is loading and it will be noticeable in non localhost env.
All error from this request is not handled here

and also remove `initialTable` prop all together